deployments/worldchain.diamond.json

```diff
@@ -0,0 +1,76 @@
+{
+    "LiFiDiamond": {
+        "Facets": {
+            "0x57FdfF2e36De6c8a8Cde297B150Ae291132Eae8d": {
+                "Name": "DiamondCutFacet",
+                "Version": "1.0.0"
+            },
+            "0xA1a4d577709dC4A70CA38F1D41562fab3aD09D3f": {
+                "Name": "DiamondLoupeFacet",
+                "Version": "1.0.0"
+            },
+            "0x81Ae738700D8f1e5BB2A200584174cDf17Fb5455": {
+                "Name": "OwnershipFacet",
+                "Version": "1.0.0"
+            },
+            "0xfA009cd56d35AE3BbdF975135b0BAE9b403c7da1": {
+                "Name": "WithdrawFacet",
+                "Version": "1.0.0"
+            },
+            "0xE154389c1bAE241F220661131b7AfDc1514C55c7": {
+                "Name": "DexManagerFacet",
+                "Version": "1.0.0"
+            },
+            "0x314E1E760316050B0D6338bCf3d689b8D301F593": {
+                "Name": "AccessManagerFacet",
+                "Version": "1.0.0"
+            },
+            "0x54ECfbAaeb49c864a9c45C70B785ca6C70c66453": {
+                "Name": "PeripheryRegistryFacet",
+                "Version": "1.0.0"
+            },
+            "0x14Dd70456Bfe4Cd8b605f7A0d24b3A74aCe99713": {
+                "Name": "LIFuelFacet",
+                "Version": "1.0.0"
+            },
+            "0xd2B3b3605e630232c13111458Ae3a97d13c8F477": {
+                "Name": "GenericSwapFacet",
+                "Version": "1.0.0"
+            },
+            "0xBa713B18c806EcdEEE49FAec623dE2D872192872": {
+                "Name": "GenericSwapFacetV3",
+                "Version": "1.0.0"
+            },
+            "0xe6C6A35684308f2DaadbeeA50B62CFEaAFaa407E": {
+                "Name": "StandardizedCallFacet",
+                "Version": "1.0.0"
+            },
+            "0x1feB868BF64AdC552E051fB7387681F78b988a81": {
+                "Name": "CalldataVerificationFacet",
+                "Version": "1.0.0"
```

Comment:
Just noticed that we have the outdated version of this facet deployed here....but I guess we can update it when we deploy the new version to all chains...